Makefile

```diff
@@ -68,13 +77,20 @@ shellcheck: ## Lints Bash and sh scripts.
 # don't maintain those scripts. Omitting the `.venv/` dir because we don't control
 # files in there. Omitting the ossec packages because there are a LOT of violations,
 # and we have a separate issue dedicated to cleaning those up.
+	@docker create -v /mnt --name shellcheck-targets circleci/python:2 /bin/true
```

Comment:
This was failing on me for subsequent runs because docker will throw an error if you try to create a new container that already exists. I ended up doing:

```diff
@@ -77,7 +77,7 @@ shellcheck: ## Lints Bash and sh scripts.
 # don't maintain those scripts. Omitting the `.venv/` dir because we don't control
 # files in there. Omitting the ossec packages because there are a LOT of violations,
 # and we have a separate issue dedicated to cleaning those up.
-       @docker create -v /mnt --name shellcheck-targets circleci/python:2 /bin/true
+       @docker create -v /mnt --name shellcheck-targets circleci/python:2 /bin/true 2> /dev/null || true
        @docker cp $(PWD)/. shellcheck-targets:/mnt/
        @find "." \( -path "./.venv" -o -path "./install_files/ossec-server" \
                -o -path "./install_files/ossec-agent" \) -prune \
```

to fix that scenario